code/game/objects/structures/grille.dm

```diff
@@ -158,6 +158,8 @@
 			icon_state = "grille"
 			R.use(1)
 			return
+	else if(istype(W, /obj/item/weapon/rcd) && istype(loc, /turf/simulated)) //Do not attack the grille if the user is holding an RCD

```

Comment:
I should probably also prevent airlocks from being used when deconstructed, but isn't `attack()` used only for living mobs? As far as I can tell there is `attack(mob/M as mob, mob/user as mob)` for creatures which is called from the attacking item and `attackby(obj/item/weapon/W as obj, mob/user as mob, params)` for objects and structures which is called from the object being attacked.